lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -30,6 +30,10 @@
 
 #include <numeric>
 
+// Remove later: KD
```

Comment:
Remove debugging?

Also, most llvm types print better with errs() or dbgs() vs iostream. They come from llvm/Support/raw_ostream.h

Reply:
Sorry, had missed that. Removed. And, will use errs() and dbgs() going forward.